src/services/api.service.ts

```diff
@@ -66,6 +66,11 @@ export const updateCourse = async (
   return data;
 };
 
+export const updateLectures = async (lectures: Lecture[]): Promise<Course> => {
+  const { data } = await axiosInstance().put(`/lectures`, lectures);
```

Comment:
I do not think we have an endpoint that accepts an array of lectures. We will need to add such endpoint to the back-end first

Reply:
I dit it but forgot to push it